presto-yarn-package/src/main/slider/package/scripts/configure.py

```diff
@@ -34,8 +34,13 @@ def set_configuration(component=None):
     _directory(params.log_dir, params)
 
     _template_config("{params.conf_dir}/config.properties", params, component)
-    _template_config("{params.conf_dir}/jvm.config", params)
     _template_config("{params.conf_dir}/node.properties", params)
+    
+    if params.jvm_args:
+        jvm_arg_list = ast.literal_eval(params.jvm_args)
+        for arg in jvm_arg_list:
+            with open(format("{params.conf_dir}/jvm.config"), 'a') as fw:
+                fw.write("%s\n" % arg)
```

Comment:
open file once per loop do not use append mode in case there were already such file